bug fixes in bioconductor_docker, rebuild stack

nturaga opened the pull request:
There were bug fixes for a package in bioconductor/bioconductor_docker:RELEASE_3_13. So we need to rebuild the stack on AnVIL

nturaga committed:
bug fixes in bioconductor_docker, rebuild stack

There were bug fixes for a package in bioconductor/bioconductor_docker:RELEASE_3_13. So we need to rebuild the stack on AnVIL

nturaga committed:
bug fixes in bioconductor_docker, rebuild stack

There were bug fixes for a package in bioconductor/bioconductor_docker:RELEASE_3_13. So we need to rebuild the stack on AnVIL

rtitle commented on anvil-rstudio-bioconductor/CHANGELOG.md:
```diff
@@ -1,3 +1,11 @@
+## 3.13.1 - 6/14/2021
```
I think this should be 3.13.2? I think we already pushed 3.13.1 for a bugfix.

nturaga replied:
Apologies for the confusion @rtitle . I think it's all straight now.

nturaga replied:
Or maybe I messed it all up.....trying to fix for 3.13.1 and 3.13.2 ...any help is appreciated...or I purge the PR and redo

nturaga committed:
Version 3.13.2

- Update README.md to include 3.13.1 version as well on 6/10.

- Update README to include 3.13.2

nturaga committed:
Update version"

nturaga committed:
Merge branch 'master' into bug_fix_libeigen
